samcli/local/apigw/local_apigw_service.py

```diff
@@ -455,3 +468,26 @@ def _should_base64_encode(binary_types, request_mimetype):
 
         """
         return request_mimetype in binary_types or "*/*" in binary_types
+
+    @staticmethod
+    def cors_to_headers(cors):
+        """
+        Convert CORS object to headers dictionary
+        Parameters
+        ----------
+        cors list(samcli.commands.local.lib.provider.Cors)
+            CORS configuration objcet
+        Returns
+        -------
+            Dictionary with CORS headers
+        """
+        if not cors:
+            return {}
+        headers = {
+            'Access-Control-Allow-Origin': cors.allow_origin,
+            'Access-Control-Allow-Methods': cors.allow_methods,
+            'Access-Control-Allow-Headers': cors.allow_headers,
+            'Access-Control-Max-Age': cors.max_age
+        }
+
+        return {h_key: h_value for h_key, h_value in headers.items() if h_value is not None}
```

Comment:
What is this doing? This is hard for me to read.

Reply:
It's taking the cors_headers dict and filtering out properties from Cors object that don't exist.